src/ResourceManager/ServiceFabric/Commands.ServiceFabric/Commands/NewAzureRmServiceFabricCluster.cs

```diff
@@ -393,7 +393,7 @@ private void DeployWithDefaultTemplate()
                 }
                 else
                 {
-                    WriteVerbose(string.Format("Found existing cluster {0} which's status is waiting for nodes", this.Name));
+                    WriteVerboseWithTimestamp(string.Format("Found existing cluster {0} which's status is waiting for nodes", this.Name));
```

Comment:
In a future release, this should become a resource string, since it is displayed to the user